chakracore/10/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 1.12.3
+ENV YARN_VERSION 1.21.1
```

Comment:
Do we used to update image by Yarn version change only?

Reply:
It's a security release for yarn as well. The fact there is no new chakra release of node (and this no npm update) doesn't mean we shouldn't update yarn in it, imo

Reply:
I agree we should figure a way to provide an update for that but for the pass cases if I didn't recalled wrong we didn't update image only triggered by Yarn update, so maybe we'll need more discussion here.

Reply:
I disagree - it's a security release and should be updated regardless of whether or not the node version changed. I _highly_ doubt this image sees much use anyways, the last commit to the chakra node repo came more than a year ago